onnx/numpy_helper.py

```diff
@@ -13,6 +14,12 @@ def combine_pairs_to_complex(fa: Sequence[int]) -> Sequence[np.complex64]:
     return [complex(fa[i * 2], fa[i * 2 + 1]) for i in range(len(fa) // 2)]
 
 
+# convert ndarray of bf16 (as uint32) to f32 (as uint32)
+def bfloat16_to_float32(data: np.ndarray, dims: nptyping._ShapeLike) -> np.ndarray:
+    shift = lambda x: x << 16
```

Comment:
https://github.com/python/mypy/issues/9590 Mypy might have some issues with lambda... I am still investigating. Will keep you posted.

Reply:
I am not sure why mypy will complain this lambda. It seems like a false positive to me and it still exists in the latest mypy. You can just simply use `# type: ignore` to ignore it for now. Or, shift the variable without using lambda.